react-native.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
   env: {
     'react-native/react-native': true,
   },
-  plugins: ['react-native'],
+  plugins: ['react-native', '@react-native'],
   rules: {
+    '@react-native/platform-colors': WARNING,
```

Comment:
can we turn on all recommended rules?

Reply:
`@react-native` plugin is exported only one rule )